solr/server/build.gradle

```diff
@@ -111,7 +112,12 @@ task assemblePackaging(type: Sync) {
     into "lib/ext"
   })
 
-  from { project.configurations.startJar.singleFile } {
+  from("${buildDir}/libs", {
```

Comment:
This doesn't seem to be right. you rename twice into the same solr.jar. And I don't think it's elegant that the server depends on the launcher (jetty-start). It's a smell.

Reply:
It's just a hack so far to get it building. I thought the bootstrap class belongs in server module, but there is probably a better way to copy the lib in place. Or we could have a completely new solr/bootstrap module with its own build file just to produce solr.jar?

Reply:
I think a new module would make more sense (put a comment in jira).